src/controllers/handlers/scene-stream-access-handlers/add-scene-stream-access-handler.ts

```diff
@@ -45,10 +45,14 @@ export async function addSceneStreamAccessHandler(
     place = await getPlaceByParcel(parcel)
   }
 
-  const canCreateStreamKey = await isSceneOwnerOrAdmin(place, authenticatedAddress)
+  const authenticatedUserScenePermissions = await resolveUserScenePermissions(place, authenticatedAddress)
 
-  if (!canCreateStreamKey) {
-    logger.info(`Wallet ${authenticatedAddress} is not owner nor admin of the scene. Place ${place.id}`)
+  if (
```

Comment:
This check is repeated multiple times in the files, what do you think if we create a new function, called `isSceneOwnerOrAdmin` (like we had before) that uses `resolveUserScenePermissions`?